AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -407,7 +407,7 @@ class CallingContext: NSObject {
 extension CallingContext: CallDelegate {
     func call(_ call: Call, didChangeState args: PropertyChangedEventArgs) {
         switch call.state {
-        case .none:
+        case .none, .disconnected:
```

Comment:
Here is for the scenario that the user is kicked out or being denied in the Teams meeting. 
If the call state has been changed in to `disconnected`, do we still need to check `none` state?

Reply:
@Gemakk I noticed when leaving the call will also be in disconnected state. So reverting back the code and created a bug for SDK team https://github.com/Azure/Communication/issues/313

Reply:
I also created a bug in our board. We need to do some code investigation and update in the Sample App. 
https://skype.visualstudio.com/SPOOL/_workitems/edit/2468223